### PR TITLE
adding slide info

### PR DESCRIPTION
**Pull request by carriewright11**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -75,6 +75,28 @@ TODO: Can have github actions run this for us.
...
+
+Please select the layout that looks like this for any content from outside sources:
+
+![](https://docs.google.com/presentation/d/1-7UvgVq5tP1pasTEErUM3bJFH2fU_pilH6i6_81CCXU/edit#slide=id.gcf0c1d8548_0_146)
+
+Please update the text at the bottom to describe the source.
+
+For any major point, please select this layout:
+
+![](https://docs.google.com/presentation/d/1-7UvgVq5tP1pasTEErUM3bJFH2fU_pilH6i6_81CCXU/edit#slide=id.gcf0c1d8548_0_136)
```
Comment (cansavvy) [inline CONTRIBUTING.md]: Doesn't look like these links are rendering properly

**Comment by cansavvy** [comment]
This looks good to me except that the google slide links aren't rendering as far as I can tell.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -75,6 +75,28 @@ TODO: Can have github actions run this for us.
...
 
+### Adding Images and Graphics
+
+To maintain style and attributions for graphics and images, as well as to enable easy updates in the future, please start a new Google Slide document for your course and import this the theme from this [template](https://docs.google.com/presentation/d/1-7UvgVq5tP1pasTEErUM3bJFH2fU_pilH6i6_81CCXU/edit?usp=sharing).
+
+See [this video](https://youtu.be/pNbwF263yY8) for assistance on how to import themes. 
+
+Please select the layout that looks like this for any content from outside sources:
+
+![](https://docs.google.com/presentation/d/1-7UvgVq5tP1pasTEErUM3bJFH2fU_pilH6i6_81CCXU/edit#slide=id.gcf0c1d8548_0_146)
```
Comment (cansavvy) [inline CONTRIBUTING.md]: This allows the google slide to be previewed, but unfortunately doesn't zero in on one slide.
```suggestion
<style>
.responsive-wrap iframe{ max-width: 100%;}
</style>
<div class="responsive-wrap">
<!-- this is the embed code provided by Google -->
  <iframe src="https://docs.google.com/presentation/d/1-7UvgVq5tP1pasTEErUM3bJFH2fU_pilH6i6_81CCXU/edit#slide=id.gcf0c1d8548_0_146/" frameborder="0" width="960" height="569" allowfullscreen="true" mozallowfullscreen="true" ></iframe>
<!-- Google embed ends -->
</div>
```

Comment (cansavvy) [reply CONTRIBUTING.md]: Alternatively, using "publish to web" described here might work? https://help.edublogs.org/google-slides/ And then you could include the actual regular link to get there.

Comment (carriewright11) [reply CONTRIBUTING.md]: There is a way to do it where you use a different link like this: https://www.labnol.org/internet/direct-links-for-google-drive/28356/ 
I always find it tricky though and was struggling to make it work - I'm still working on it.

Comment (carriewright11) [reply CONTRIBUTING.md]: something like this: https://developers.google.com/slides/samples/reading

Comment (carriewright11) [reply CONTRIBUTING.md]: https://www.evernote.com/shard/s425/sh/fd22f17a-5b1e-c3d6-eb85-e47c20aa4b1d/58951432cb4f6f996ad9a28df6852248

Comment (cansavvy) [reply CONTRIBUTING.md]: Awesome! This looks good! The first slide being blank kinda makes it seem like its not working, but after you scroll down its clear it is working.

Comment (carriewright11) [reply CONTRIBUTING.md]: yeah - we can rearrange them and that might look better or maybe I should add an arrow pointing to the text at the bottom

Comment (cansavvy) [reply CONTRIBUTING.md]: Either idea sounds good!

**Review by cansavvy — APPROVED** [review]
This looks good to me! Is it ready?

**Comment by carriewright11** [comment]
Yup, it is ready. Was just waiting for you to see it.

**Pull request merged**